backend/config/backend.go

```diff
@@ -21,12 +21,19 @@ type WebSocketConfig struct {
 	WriteBufferSize int `json:"write_buffer_size"`
 }
 
+type HttpConfig struct {
+	AllowOrigin  []string `json:"allow_origin"`
+	AllowMethods []string `json:"allow_methods"`
+	AllowHeaders []string `json:"allow_headers"`
+}
+
 type BackendConfig struct {
-	Host       string               `json:"host"`
-	Port       int                  `json:"port"`
-	Scripts    BackendScriptsConfig `json:"scripts"`
-	Production bool                 `json:"production"`
-	WebSocket  WebSocketConfig      `json:"websocket"`
+	Host             string               `json:"host"`
+	Port             int                  `json:"port"`
+	Scripts          BackendScriptsConfig `json:"scripts"`
+	Production       bool                 `json:"production"`
+	WebSocket        WebSocketConfig      `json:"websocket"`
+	HttpConfigValues HttpConfig           `json:"http_config"`
```

Comment:
Kinda a nit, but I think "HttpConfig" or "Http" would be a better name than "HttpConfigValues"